Zotero/Controllers/Database/Requests/EditItemFromDetailDbRequest.swift

```diff
@@ -45,10 +45,8 @@ struct EditItemFromDetailDbRequest: DbRequest {
     }
 
     private func updateCreators(with data: ItemDetailState.Data, snapshot: ItemDetailState.Data, item: RItem, changes: inout RItemChanges, database: Realm) {
-        guard data.creatorIds != snapshot.creatorIds else { return }
-
+        guard data.creators != snapshot.creators else { return }
```

Comment:
I  removed the older condition and added this one because I noticed when creators are changed we remove all creators and create new ones. This one covers all cases and so comparing the IDs may not be necessary.

Reply:
My bad, I didn't notice you added another condition. All good.